dynomite/tests/integration/main.rs

```diff
@@ -0,0 +1,10 @@
+// With this `macro_use` we are able to use standard `assert_eq/assert_ne`
```

Comment:
Moved this into a single integration tests as per [this article](https://matklad.github.io/2021/02/27/delete-cargo-integration-tests.html)